src/transformers/models/mctc/processing_mctc.py

```diff
@@ -0,0 +1,73 @@
+# coding=utf-8
+# Copyright 2021 The HuggingFace Inc. team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Speech processor class for MCTC
+"""
+from contextlib import contextmanager
+
+from ...processing_utils import ProcessorMixin
+
+
+class MCTCProcessor(ProcessorMixin):
+    r"""
+    Constructs a MCTC processor which wraps a MCTC feature extractor and a MCTC tokenizer into a single processor.
+
+    [`MCTCProcessor`] offers all the functionalities of [`MCTCFeatureExtractor`] and [`MCTCTokenizer`]. See the
+    [`~MCTCProcessor.__call__`] and [`~MCTCProcessor.decode`] for more information.
+
+    Args:
+        feature_extractor (`MCTCFeatureExtractor`):
+            An instance of [`MCTCFeatureExtractor`]. The feature extractor is a required input.
+        tokenizer (`MCTCTokenizer`):
+            An instance of [`MCTCTokenizer`]. The tokenizer is a required input.
+    """
+    feature_extractor_class = "MCTCFeatureExtractor"
+    tokenizer_class = "MCTCTokenizer"
```

Comment:
```suggestion
    tokenizer_class = "AutoTokenizer"
```

I don't think we need a new tokenizer here actually - see longer comment below